core/iwasm/fast-jit/fe/jit_emit_memory.c

```diff
@@ -203,36 +211,42 @@ jit_compile_op_i32_load(JitCompContext *cc, uint32 align, uint32 offset,
     }
 
     memory_data = get_memory_data_reg(cc->jit_frame, 0);
+#if WASM_ENABLE_SHARED_MEMORY != 0
+    if (atomic) {
+        addr = jit_cc_new_reg_I64(cc);
+        CHECK_ALIGNMENT(addr, memory_data, offset1);
```

Comment:
Only need to check the alignment of offset1, no need to `GEN_INSN(ADD, maddr, memory_data, offset1)` in CHECK_ALIGNMENT, since memory_data must be 8-byte aligned.
Had better change the original CHECK_ALIGNMENT and original check in atomic wait, and here move the check to be after L211.